tests/__init__.py

```
@@ -190,14 +190,13 @@ def emit(self, record: logging.LogRecord) -> None:
-			raise ImportError(str("[CWE-758] Test module failed completely.")) from impErr
-except Exception as badErr:  # pragma: no branch
-	_LOGGER.debug(str(type(badErr)))
-	_LOGGER.exception(str(badErr))
-	_LOGGER.debug(str((badErr.args)))
-	badErr = None
-	del badErr  # skipcq - cleanup any error leaks early
+		except Exception as _root_cause:  # pragma: no branch
+			raise ImportError(str("[CWE-758] Test module failed completely.")) from _root_cause
+except Exception as _cause:  # pragma: no branch
+	_LOGGER.debug(str(type(_cause)))
+	_LOGGER.exception(str(_cause))
```

Ruff (0.8.2)

Redundant exception object included in `logging.exception` call

(TRY401)